docs/source/release/v6.10.0/framework.rst

```diff
@@ -10,70 +10,128 @@ Algorithms
 
 New features
 ############
-.. amalgamate:: Framework/Algorithms/New_features
+- Algorithm :ref:`algm-SaveASCII` now supports 1D :ref:`MDHistoWorkspace`.
+- Algorithm :ref:`algm-CompressEvents` is now able to combine events together in logarithmic increasing size groups.
+- Algorithm `PolarizationCorrectionWildes` now accepts input workspace order.
+- Algorithms :ref:`algm-Abins` and :ref:`algm-Abins2D` can now import phonopy .yaml data.
+  The force constants are stored in a file named ``FORCE_CONSTANTS`` or
+  ``force_constants.hdf5`` in the same directory as the YAML file.
+  This is recommended when using large force constants arrays as the
+  YAML loader can be very slow.
```

Comment:
The spacing of the bullet points in this section is wider once built. Not a critical issue, could be to do with the 
```
The Euphonic JSON formats are convenient to create with Python
scripts, and recommended for users who wish to somehow customise or
manipulate their data before using it with Abins(2D).
```
text block?

Reply:
Yeah, it was caused by that extra indentation